packages/generators/add-generator.ts

```diff
@@ -372,39 +409,40 @@ module.exports = class AddGenerator extends Generator {
 						}
 					} else {
 						// If its not in webpack, check npm
-						npmExists(answerToAction.actionAnswer).then(p => {
-							if (p) {
-								this.dependencies.push(answerToAction.actionAnswer);
-								const normalizePluginName = answerToAction.actionAnswer.replace(
-									"-webpack-plugin",
-									"Plugin"
-								);
-								const pluginName = replaceAt(
-									normalizePluginName,
-									0,
-									normalizePluginName.charAt(0).toUpperCase()
-								);
-								this.configuration.config.topScope.push(
-									`const ${pluginName} = require("${
-										answerToAction.actionAnswer
-									}")`
-								);
-								this.configuration.config.webpackOptions[
-									action
-								] = `new ${pluginName}`;
-								this.configuration.config.item = answerToAction.actionAnswer;
-								done();
-								this.runInstall(getPackageManager(), this.dependencies, {
-									"save-dev": true
-								});
-							} else {
-								console.error(
-									answerToAction.actionAnswer,
-									"doesn't exist on NPM or is built in webpack, please check for any misspellings."
-								);
-								process.exit(0);
-							}
-						});
+						npmExists(answerToAction.actionAnswer)
+							.then((p: string) => {
+								if (p) {
```

Comment:
missing ret on npmexists

Reply:
`npmExists` returns a Promise whose types are added in _/utils_ PR. This should be implicitly covered when we merge all PRs. :)